plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -730,6 +730,21 @@ class FeatureDevController(
                 messageType = FeatureDevMessageType.Answer,
                 message = message("amazonqFeatureDev.follow_up.modified_source_folder", selectedFolder.path),
             )
+
+            messenger.sendAnswer(
+                tabId = tabId,
+                messageType = FeatureDevMessageType.SystemPrompt,
+                followUp = listOf(
+                    FollowUp(
+                        pillText = message("amazonqFeatureDev.follow_up.retry"),
+                        type = FollowUpTypes.RETRY,
```

Comment:
would it make more sense for the retry to be automatic after they select a new folder?